docs/01_nodeos/03_plugins/trace_api_plugin/index.md

```diff
@@ -0,0 +1,115 @@
+# trace_history_plugin
+
+## Description
+
+The `trace_api_plugin` provides a consumer-focused long-term API for retrieving blocks containing retired actions and related metadata.
+
+## Usage
```

Comment:
Somewhere we should indicate the new http endpoint `/v1/trace_api/get_block`. See trace_api_plugin PR for more info.

Reply:
Got it, will add here. It will also be covered in the RPC API reference (PR coming shortly).

Reply:
Ah, ok if in there then that is fine. Just thought we needed it somewhere.